polaris/polaris/templates/transaction/more_info.html

```diff
@@ -88,22 +105,22 @@
                 return;
             }
 
-            // Anchors should send a JSON with the the entire transaction object, serialized by
-            // the server, in a transaction property (i.e., the value for the "transaction" key).
-            // The transaction object should not be hand-constructed at this point.
-            // The reference client expects the following fields in the transaction: id, status,
-            // withdraw_anchor_account, withdraw_memo, withdraw_memo_type, amount_in.
-            //
-            // In other terms, the message should have the following structure:
-            // tx_json = {
-            //     "transaction": {
-            //         "id": "fake-transaction-id",
-            //         "status": "completed"
-            //         etc.
-            //     }
-            // }
             targetWindow.postMessage(tx_json, "*");
         }
+
+        // Callback function to post the serialized transaction to the callback URL
+        function urlCallback() {
```

Comment:
i think this should be done server side, like we talked about this morning then we can have immediate callbacks regardless of whether this page is open in the client or not.

Reply:
looks like this actually matches the sep in that it only communicates that the interactive flow is finished, so np here